tests/TestCase/ORM/EntityTest.php

```diff
@@ -27,6 +29,31 @@ class EntityTest extends TestCase
 {
 
     /**
+     * Fixtures to be used
+     *
+     * @var array
+     */
+    public $fixtures = ['core.authors'];
+
+    /**
+     * Tests that the source of an existing Entity is the same as a new one
+     *
+     * @return void
+     */
+    public function testEntitySourceExistingAndNew()
+    {
```

Comment:
I would put this test in the table suite as that suite already has fixtures and adding a fixture here will impact all the tests performance wise.